### ISSUE TITLE
sortedimports: code between imports gets deleted

### ISSUE THREAD

**Issue by knlr** [nicklockwood/SwiftFormat#235]
My project wouldn't compile after running swiftformat because it deleted some code. It turned out that the sortedimports rule assumes all the imports are in one place, and it will discard anything in between the first and last 'import' that isn't an import:

```
import Foo

func bar() {} //<- will get deleted

import Bar
```

**Comment by nicklockwood** [comment]
Thanks, I'll include a fix in the next release.

**Comment by nicklockwood** [comment]
@knlr fixed in 0.33.5
